tests/functional/cylc-combination-scripts/03-vr-resume.t

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#------------------------------------------------------------------------------
+# Test `cylc vr` (Validate Reinstall restart)
+# In this case the target workflow is paused so cylc reload & cylc play are run.
```

Comment:
> In this case the target workflow is paused so cylc reload & cylc play are run.

`cylc vr` does not distinguish between running and paused workflows and does not run `cylc play` on them.

Reply:
I'll add that to 01 - I've deleted this on my branch.